### PR TITLE
Docs Information Architecture overhaul

### PR DESCRIPTION
**Pull request by leggetter**
More info to come. Here's what it looks like, and feel free to have play with the preview once it's been built.

## Before

image

## After

image

## Walkthrough

Note: SDKs & Tools section was added after this video was created which splits References. See https://github.com/tigrisdata/tigris-docs/pull/278/commits/35fe48a28a4f8e9ffb1fd4822411fba2ffd9332c

https://user-images.githubusercontent.com/328367/213181983-f613aabe-7f9a-4f7a-aeb9-f0ffbafdd3d8.mp4

---

Note: basic redirects seem to work e.g. https://docs-git-dx-updates-tigrisdata.vercel.app/docs/typescript/ -> https://docs-git-dx-updates-tigrisdata.vercel.app/docs/typescript/

However, the sub-paths aren't being captured e.g. https://docs-git-dx-updates-tigrisdata.vercel.app/docs/typescript/getting-started/

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -1,3 +1,105 @@
...
-  "trailingSlash": true
+  "trailingSlash": true,
+  "redirects": [
+    {
+      "source": "/docs/overview/",
+      "destination": "/docs/concepts/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/overview/",
```
Comment (ovaistariq) [inline vercel.json:10]: Duplicate of above

**Comment by ovaistariq** [comment]
Looks great overall. The only change I would recommend is breaking out `References` into `SDKs` and `API Reference`

File: docs/quickstarts/quickstart.mdx [hunk 2]
```diff
@@ -22,15 +23,14 @@ need to step out of your application code.
...
+To sign up for Tigris, go to the <a href={tigrisConfig.signupUrl}>Tigris Cloud console</a>.
 You can choose to authenticate with Google, GitHub, or by using an email.
 
 ## Create a project
 
 Once you have successfully signed up, you are ready to start creating a project.
 You can create a new project from the Tigris Cloud dashboard by clicking on the
-[Create a new project button](https://console.preview.tigrisdata.cloud/) and
-providing a project name.
+Create a new project button and providing a project name.
```
Comment (JigarJoshi) [inline docs/quickstarts/quickstart.mdx:33]: nit: maybe quote or back tick "Create a new project" to differentiate it as the button title.

**Comment by JigarJoshi** [comment]
nit: when I go to sdk & tools (https://docs-git-dx-updates-tigrisdata.vercel.app/docs/sdkstools/ as per last deployed url) I see 

image

IMO - instead of 2 items, a small description on what it contains will be more helpful.

**Comment by JigarJoshi** [comment]
Overall restructuring is looking great 👍

**Comment by leggetter** [comment]
> nit: when I go to sdk & tools (https://docs-git-dx-updates-tigrisdata.vercel.app/docs/sdkstools/ as per last deployed url) I see
> 
> image
> 
> IMO - instead of 2 items, a small description on what it contains will be more helpful. like previous view
> 
> image

Agreed. Looks like we'd need to [swizzle](https://docusaurus.io/docs/swizzling) the `DocCardList` and customize.

File: docs/quickstarts/quickstart.mdx (hunk 2 above)
Comment (leggetter) [reply docs/quickstarts/quickstart.mdx]: 👍 bolded in https://github.com/tigrisdata/tigris-docs/pull/278/commits/9f09faa8a0e53439171a1466ab181fe882db8c4b

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**